app/cdap/components/ConnectionList/Components/TabLabelCanSample/index.tsx

```diff
@@ -17,30 +17,29 @@
 import { Typography } from '@material-ui/core';
 import Box from '@material-ui/core/Box';
 import CustomTooltip from 'components/ConnectionList/Components/CustomTooltip';
+import useStyles from 'components/ConnectionList/Components/TabLabelCanSample/styles';
```

Comment:
why is useStyles exported as default
plz use export const useStyles

Reply:
Done

Reply:
Done